tests/testUtilities/testUtils.ts

```diff
@@ -140,6 +140,7 @@ export interface ZoneData {
 
 export interface AttenuationData {
```

Comment:
So, does the returned data usually include "id", or is it "zone-attenuation-id"?